src/renderer/components/editorWithTabs/sourceCode.vue

```diff
@@ -245,6 +261,49 @@ export default {
         this.tabId = null // invalidate tab id
       }
     },
+    handleUndo () {
```

Comment:
Please keep spacing (new lines) between methods. You could group e.g. undo/redo etc if you like.